contents/csharp/Xamarin.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Xamarin
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Xamarin
+
+**Author: [Ni Tianzhen](https://github.com/niqiukun)**
+
+<box id="article-toc">
+
+* [What is Xamarin‎](#what-is-xamarin)
+* [How Xamarin Works?](#how-xamarin-works)
+    * [Xamarin.Android and Xamarin.iOS](#xamarin.android-and-xamarin.ios)
+    * [Xamarin.Forms](#xamarin.forms)
+    * [Xamarin.Essentials](#xamarin.essentials)
+* [Why Xamarin?‎](#why-xamarin)
+    * [Complete Binding for Underlying SDKs](#complete-binding-for-underlying-sdks)
+    * [Objective-C, Java, C, and C++ Interop](#objective-c-java-c-and-c-interop)
+    * [Modern Language Constructs and Robust Base Class Library](#modern-language-constructs-and-robust-base-class-library)
+    * [Mobile Cross-Platform Support](#mobile-cross-platform-support)
+    * [Modern Integrated Development Environment (IDE)](#modern-integrated-development-environment-ide)
+* [How to Get Started‎?](#how-to-get-started)
+</box>
+
+## What is Xamarin?
+
+**Xamarin** is an open source developer platform for building modern and performant applications on various platforms, especially Android and iOS. It is built on top of Mono, an open-source version of the .NET Framework based on the .NET ECMA standards. It is currently maintained by Microsoft and the .NET community. As one of the most established mobile app development frameworks, Xamarin offers developers the option of writing apps in C#, instead of Java and Objective-C/Swift. The architecture of Xamarin aims to achieve maximum code reuse across different platforms while maintaining near-native performance.
+
+## How Xamarin Works?
+
+Here is an overview of the architecture of a cross-platform Xamarin application.
+
+![Architecture of Xamarin](xamarin-forms-architecture.png)
+*Figure 1. Overview of the architecture of a cross-platform Xamarin Application*
+
+Xamarin allows developers to create native UI on each platform and write business logic in C# that is shared across platforms. In most cases, 80% of application code is sharable using Xamarin. A typical cross-platform Xamarin project consists of three sub-projects: **Shared Code, Xamarin.Android, and Xamarin.iOS**.
```

Comment:
Since you bold the 3 projects I would expect a short intro for each of them in the following paragraph? seems that shared code does not have its own section

Reply:
Also the following section like Xamarin.Forms does not seems to be mentioned here.